### DISCUSSION TITLE
[Announcement] Dropping support for AEM 6.4 starting with release 3.0.0

### DISCUSSION THREAD

**Discussion by vladbailescu** [adobe/aem-core-wcm-components#1563] [General]
Since AEM 6.4 has reached end fo core support (04/30/2021), we will only support AEM 6.5 and AEMaaCS starting with release 3.0.0.

Customers using Core Components on 6.4 that require a critical bug or security fix can reach out to Customer Support and request those (which will probably be delivered as a public 2.x release). We won't be back-porting new features though.

Let us know if you have any concerns related to this!

**Comment by Kami6e** [comment]
Hi @vladbailescu, does it mean that for example the versions 2.18 and 2.19 are still compatible with AEM 6.4? 
It is not completely clear looking at this announcement and the documentation here: https://github.com/adobe/aem-core-wcm-components#system-requirements 

Thanks for clarifying!
